e2e/cypress/support/ui_commands.js

```diff
@@ -367,3 +367,22 @@ Cypress.Commands.add('updateChannelHeader', (text) => {
         type('{enter}').
         wait(TIMEOUTS.TINY);
 });
+
+// ***********************************************************
+// File Upload
+// ************************************************************
+
+/**
+ * Upload a file on target input given a filename and mime type
+ * @param {String} targetInput - Target input to upload a file
+ * @param {String} fileName - Filename to upload from the fixture
+ * @param {String} mimeType - Mime type of a file
+ */
+Cypress.Commands.add('fileUpload', (targetInput, fileName = 'mattermost-icon.png', mimeType = 'image/png') => {
```

Comment:
I think you need to sync and merge for this to go away, this code is already in master.

Reply:
done